deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml

```
@@ -212,6 +212,14 @@ objects:
     - update
     - patch
     - delete
+  - apiGroups:
+    - projctl.konflux.dev
+    resources:
+    - Project
+    - ProjectDevelopmentStreamTemplate
+    - ProjectDevelopmentStream
```

The resource names should be all lowercase and in a plural form.
```suggestion
    - projects
    - projectdevelopmentstreamtemplates
    - projectdevelopmentstreams
```

---

Since I cant push to @hmariset's fork Created PR #1004 to fix these issues